java/client/src/org/openqa/selenium/remote/codec/AbstractHttpCommandCodec.java

```diff
@@ -214,6 +219,13 @@ public AbstractHttpCommandCodec() {
     defineCommand(SWITCH_TO_CONTEXT, post("/session/:sessionId/context"));
     defineCommand(GET_CURRENT_CONTEXT_HANDLE, get("/session/:sessionId/context"));
     defineCommand(GET_CONTEXT_HANDLES, get("/session/:sessionId/contexts"));
+
+    // Cast / Media Router APIs
+    defineCommand(GET_CAST_SINKS, get("/session/:sessionId/goog/cast/get_sinks"));
```

Comment:
Because these aren't general commands for all drivers, please move these to `ChromiumDriverCommandExecutor`